engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -76,9 +78,11 @@ public Endpoint getOrCreateDefaultEmailSubscription(String accountId, String org
 
     public List<Endpoint> getTargetEndpoints(String orgId, EventType eventType) {
         String query = "SELECT DISTINCT e FROM Endpoint e JOIN e.behaviorGroupActions bga JOIN bga.behaviorGroup.behaviors b " +
-                "WHERE e.enabled IS TRUE AND b.eventType = :eventType AND (bga.behaviorGroup.orgId = :orgId OR bga.behaviorGroup.orgId IS NULL)";
+                "WHERE e.enabled IS TRUE AND e.status = :status AND b.eventType = :eventType " +
```

Comment:
From now on, only `READY` endpoints will be used while processing an incoming ingress Kafka message.